SETUP.md

```diff
@@ -34,7 +34,16 @@ nr1 nerdpack:serve
 Your terminal output should look like the following:
 ![terminal](screenshots/setup_screen04.png)
 
-4. Open Google Chrome and go to [one.newrelic.com?nerdpacks=local](https://one.newrelic.com?nerdpacks=local) (this places locally served applications in New Relic One). Click on the launcher named `Setup exercise`. You should see the following:
-![Congratulations](screenshots/setup_screen05.png)
+Congratulations! You are now set up for building New Relic One applications. Learn how to view your Setup Exercise application in the next section.
+
+## Viewing your app
```

Comment:
Moved the app-viewing content into its own section as it's not actually related to the set up instructions.